corehq/ex-submodules/phonelog/reports.py

```diff
@@ -328,9 +329,8 @@ def _create_rows(self, logs, matching_id=None):
 
         self.total_records = logs.count()
         for log in logs.order_by(self.ordering)[paged]:
-            date = str(log.date)
-            date_fmt = tz_utils.string_to_pretty_time(
-                date, self.timezone, fmt="%b %d, %Y %H:%M:%S")
```

Comment:
that's a bit odd: convert date to string -> parse date -> change timezone -> format